convert/src/label_studio_to_badgerdoc/badgerdoc_format/badgerdoc_format.py

```diff
@@ -49,9 +51,15 @@ def convert_from_labelstudio(self, labelstudio_data: LabelStudioModel):
     def convert_from_text(self, text: str):
         self.tokens_page = self.text_converter.convert(text)
 
+    def convert_from_pdf(self, pdf):
+        self.tokens_page = self.pdf_converter.convert(pdf)
+
     def export_tokens(self, path: Path) -> None:
         if self.tokens_page:
-            path.write_text(self.tokens_page.json(indent=4, by_alias=True))
+            if isinstance(self.tokens_page, list):
+                path.write_text(self.tokens_page.json(indent=4, by_alias=True))
+            else:
+                path.write_text(self.tokens_page.json(indent=4, by_alias=True))
```

Comment:
kinda funny one =)
"if" and "else" contain the same logic.
Please revisit this code again. Something's not right